eclipse-cla/src/main/java/org/eclipse/foundation/gerrit/validation/EclipseCommitValidationListener.java

```diff
@@ -211,7 +202,7 @@ public EclipseCommitValidationListener(PluginConfigFactory cfgFactory) {
 		 * Only committers can push on behalf of other users. Note that, I am asking
 		 * if the user (i.e. the person who is doing the actual push) is a committer.
 		 */
-		if (!author.getAccount().equals(user.getAccount())) {
+		if (!author.isPresent() || !author.get().getAccount().equals(user.getAccount())) {
```

Comment:
This is unnecessary as it is already tested above `!author.isPresent() || `

Reply:
Its necessary as the previous check no longer ends the processing, it adds an error message. This way it covers the case if the user doesn't have a user, or if they do and don't match the user.